superset/config.py

```diff
@@ -60,6 +60,15 @@
 # SQLALCHEMY_DATABASE_URI = 'mysql://myapp@localhost/myapp'
 # SQLALCHEMY_DATABASE_URI = 'postgresql://root:password@localhost/myapp'
 
+# In order to hook up a custom password store for all SQLACHEMY connections
```

Comment:
Better add documentation to documentation instead of config files. You can just set 
`SQLALCHEMY_CUSTOM_PASSWORD_STORE = False` here